examples/WORKSPACE.bazel

```diff
@@ -0,0 +1,47 @@
+workspace(name = "cargo_raze_examples")
```

Comment:
Why does it make it hard to review? This change shouldn't have any noticeable impact for the most part. I was trying to make it more unique/identifiable. But in looking at [the docs](https://docs.bazel.build/versions/master/skylark/lib/globals.html#workspace), it should maybe be updated to `com_github_google_cargo_raze_examples` or something(?). Any suggestions?

Reply:
I meant the file, not this line specifically, sorry. No way to add a comment on the whole file 😓 

I meant it's harder to review since Github just sees the old one as deleted and the new one as created.

Reply:
OOOOOH, because I renamed it to `WORKSPACE.bazel`! I see now! 

Below is a comparison between master and this branch. Does this help or do you still feel that I should rename the file back to `WORKSPACE`?
```diff
diff --git a/examples/WORKSPACE b/examples/WORKSPACE
index 11bac71..1005541 100644
--- a/examples/WORKSPACE
+++ b/examples/WORKSPACE
@@ -1,33 +1,47 @@
-workspace(name = "examples")
+workspace(name = "com_github_google_cargo_raze_examples")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
-    name = "io_bazel_rules_rust",
-    sha256 = "a62597502c1ee14fcc30da92fc31c69bc6b436f464939449dc7be88c76218a31",
-    strip_prefix = "rules_rust-6e5fa2c570ac2f17ac1df840d060fc8aab521a07",
+    name = "bazel_skylib",
+    sha256 = "97e70364e9249702246c0e9444bccdc4b847bed1eb03c5a3ece4f83dfe6abc44",
     urls = [
-        # Master branch as of 2020-06-06
-        "https://github.com/bazelbuild/rules_rust/archive/6e5fa2c570ac2f17ac1df840d060fc8aab521a07.tar.gz",
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz",
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.0.2/bazel-skylib-1.0.2.tar.gz",
     ],
 )
 
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
+
+bazel_skylib_workspace()
+
 http_archive(
-    name = "bazel_skylib",
-    sha256 = "9a737999532daca978a158f94e77e9af6a6a169709c0cee274f0a4c3359519bd",
-    strip_prefix = "bazel-skylib-1.0.0",
-    url = "https://github.com/bazelbuild/bazel-skylib/archive/1.0.0.tar.gz",
+    name = "io_bazel_rules_rust",
+    sha256 = "5ed804fcd10a506a5b8e9e59bc6b3b7f43bc30c87ce4670e6f78df43604894fd",
+    strip_prefix = "rules_rust-fdf9655ba95616e0314b4e0ebab40bb0c5fe005c",
+    urls = [
+        # Master branch as of 2020-07-27
+        "https://github.com/bazelbuild/rules_rust/archive/fdf9655ba95616e0314b4e0ebab40bb0c5fe005c.tar.gz",
+    ],
 )
 
-load("@io_bazel_rules_rust//:workspace.bzl", "bazel_version")
-bazel_version(name = "bazel_version")
-
 load("@io_bazel_rules_rust//rust:repositories.bzl", "rust_repositories")
+
 rust_repositories()
 
+load("@io_bazel_rules_rust//:workspace.bzl", "bazel_version")
+
+bazel_version(name = "bazel_version")
+
 load("//remote/non_cratesio/cargo:crates.bzl", "non_cratesio_fetch_remote_crates")
+
 non_cratesio_fetch_remote_crates()
 
 load("//remote/complicated_cargo_library/cargo:crates.bzl", "complicated_cargo_library_fetch_remote_crates")
+
 complicated_cargo_library_fetch_remote_crates()
 
+load("//remote/regression_test/cargo:crates.bzl", "regression_test_fetch_remote_crates")
+
+regression_test_fetch_remote_crates()
+
```

Reply:
I think it's probably worth keeping as WORKSPACE, since I don't think it provides a ton of value to rename, and it will probably break the git history

Reply:
Ok, I've restored the filename

Rant: Man, I hate `WORKSPACE` and `BUILD` so much. I really wish Bazel only supported the filenames with extensions. But oh well 😜